### PR TITLE
feature/CS-668

### PR DESCRIPTION
**Pull request by SuneetaDevi**
Made phone num clikable.

NO errors
NO warnings
Website is working fine

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -835,7 +835,9 @@ <h2>Contact Info</h2>
                                         <div class="widget-content">
                                             <a href="tel:1800-574-9687" class="contact-number">+1 647-648-2979</a>
                                             <ul>
-                                                <li>10 capreol capreol court, Toronto</li>
+                                                <li><a href="https://goo.gl/maps/cVfifCcR3DCtpKju6" target="_blank">10
+                                                        capreol capreol
```
Comment (FaizanKhalidMohsin) [inline index.html:839]: Hi @SuneetaDevi , there is still double capreol capreol. 
AND The address should be exactly: 10 Capreol Court, Toronto, M5V 4B3

File: index.html [hunk 2]
```diff
@@ -835,7 +835,9 @@ <h2>Contact Info</h2>
...
+                                                        court, Toronto</a></li>
                                                 <li>Email :<a href="mailto:info@chezBnB.com"> info@chezBnB.com</a>
```
Comment (FaizanKhalidMohsin) [inline index.html:841]: In the email address, BnB why is it capitalized? It should all be lowercase: info@chezbnb.com

**Review by FaizanKhalidMohsin — CHANGES_REQUESTED** [review]
There are two things to review. Please, complete change requests.

**Review by SuneetaDevi — COMMENTED** [review]
done with address and email

**Review by FaizanKhalidMohsin — APPROVED** [review]

**Pull request merged**